spec/plans/provision.fmf

```diff
@@ -75,3 +75,39 @@ link:
       - implemented-by: /tmt/steps/__init__.py
       - implemented-by: /tmt/queue.py
       - verified-by: /tests/multihost
+
+/ansible:
+    summary: Per-guest Ansible customization
+
+    story:
+        As a user I want to assign an Ansible group and host variables
+        for a provisioned guest to be reflected in the generated inventory.
+
+    description: |
+        Each provisioned guest may define an optional ``ansible`` key
+        to control its behavior in the generated Ansible inventory.
+
+        - ``group``: Assigns the guest to a specific group. If omitted,
+          the ``role`` value from the guest is used as the group name.
+          If no role is defined either, the host will be placed in the
+          'ungrouped' section of the inventory.
```

Comment:
What is "ungrouped" section of the inventory? Will there be guest name directly under `all`?

Reply:
I see. Makes sense. I would then speak about "group" rather than "section":

```suggestion
          ``ungrouped`` group.
```